src/Lykke.Blockchains.Integrations.Sdk/Controllers/IsAliveController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using JetBrains.Annotations;
+using Lykke.Blockchains.Integrations.Contract.Common;
+using Lykke.Blockchains.Integrations.Contract.Common.Responses;
+using Lykke.Common;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Lykke.Blockchains.Integrations.Sdk.Controllers
+{
+    [ApiController]
+    [Route("/api/isalive")]
+    [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
+    internal class IsAliveController : ControllerBase
+    {
+        [HttpGet]
+        public ActionResult<BlockchainIsAliveResponse> GetIsAlive()
+        {
+            var response = new BlockchainIsAliveResponse(
+                AppEnvironment.Name,
+                Constants.ContractVersion,
+                AppEnvironment.EnvInfo,
+                typeof(BlockchainIsAliveResponse).Assembly.GetName().Version);
```

Comment:
We should return app version, but not SDK assembly version, shouldn't we?

Reply:
Correct. It's was fixed in the next PRs. Moved to this PR.